configuration/pih/htmlforms/ncd-adult-followup.xml

```diff
@@ -689,7 +689,7 @@
 
                                 <includeIf velocityTest="$pt_has_rf or $pt_has_scd">
                                     <tr>
-                                        <obsgroup groupingConceptId="PIH:CONJUNCTIVA">
+                                        <obsgroup groupingConceptId="PIH:12657">
```

Comment:
This isn't correct.  This is "Dietary intake" and not a class="ConvSet".  What are you trying to do here @tommyiversonj